handel/src/services/ecs/cluster-auto-scaling.ts

```diff
@@ -150,11 +83,8 @@ export async function createDrainingLambdaIfNotExists(accountConfig: AccountConf
  *
  * This function is used for both 'min' and 'max' auto-scaling group calculations.
  */
-export function getInstanceCountForCluster(instanceType: string, autoScaling: HandlebarsEcsTemplateAutoScaling, containerConfigs: HandlebarsEcsTemplateContainer[], calculationType: string, serviceName: string): number {
-    const instanceMemory = EC2_INSTANCE_MEMORY_MAP[instanceType];
-    if (!instanceMemory) {
-        throw new Error(`${serviceName} - Unhandled instance type specified: ${instanceType}`);
-    }
+export async function getInstanceCountForCluster(instanceType: string, autoScaling: HandlebarsEcsTemplateAutoScaling, containerConfigs: HandlebarsEcsTemplateContainer[], calculationType: string, serviceName: string): Promise<number> {
+    const instanceMemory = await getMemoryForInstance(instanceType);
```

Comment:
This call takes a little bit of time, and we call this `getInstanceCountForCluster` twice for the same instance type. We should probably call this getMemoryForInstance() outside this function and pass the value it returns into both invocations of this function. That will result in a small speed-up at run-time.